server.go

```diff
@@ -236,7 +237,9 @@ func OptServerClusterHasher(h Hasher) ServerOption {
 
 func OptServerTranslateFileMapSize(mapSize int) ServerOption {
 	return func(s *Server) error {
-		s.holder.translateFile = NewTranslateFile(OptTranslateFileMapSize(mapSize))
+		s.holder.translateFile = NewTranslateFile(
+			OptTranslateFileMapSize(mapSize),
+			OptTranslateFileLogger(s.logger))
```

Comment:
this works, but it creates a dependency on the order in which server options are passed to NewServer. It would probably be better to save the map size here and call NewTranslateFile in NewServer after the options have been applied to the server.

Reply:
I guess the success criteria of this ticket is faulty :)